classes/userPrivateNote/UserPrivateNote.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * @file classes/user/UserPrivateNote.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class UserPrivateNote
+ *
+ * @ingroup user
+ *
+ * @see DAO
+ *
+ * @brief Basic class describing user private note existing in the system.
+ */
+
+namespace PKP\userPrivateNote;
+
+use PKP\core\DataObject;
+
+class UserPrivateNote extends DataObject
+{
+	/**
+	 * Get private note context ID.
+     *
+	 * @return int
+	 */
+    public function getContextId(): int
```

Comment:
Indentation is a bit garbled here; there should be a git pre-commit hook that applies automatic formatting. Can you check if that's working?